### PR TITLE
Updated Dockerfiles for RHEL and CentOS

### PR DESCRIPTION
**Pull request by InfoSec812**
#### What is this PR About?
Updated Dockerfiles to build NPM Jenkins slaves with NodeJS 10 and NPM 6.4.1

#### How do we test this?
```
docker build -t jenkins-slave-npm-rhel7 -f Dockerfile.rhel7 .
docker run -i jenkins-slave-npm-rhel7 npm --version

docker build -t jenkins-slave-npm-centos7 .
docker run -i jenkins-slave-npm-centos7 npm --version 
```

### Additional Details
Instead of using Software Collections in RHEL, the RHEL Dockerfile uses the Red Hat OpenShift Application Runtimes (RHOAR) repositories for installing Node/NPM. This makes the whole container simpler and less error-prone.

These containers also include tools to allow for `npm audit` to generate reports and break builds when desired.

Finally, I have included the SonarQube scanner application

cc: @redhat-cop/containerize-it

### CODE REVIEW

File: jenkins-slaves/jenkins-slave-npm/Dockerfile [hunk 1]
```diff
@@ -23,4 +20,8 @@ RUN yum install -y centos-release-scl-rh && \
     yum clean all -y && \
     rm -rf /var/cache/yum
 
-USER 1001
+USER root
+RUN npm install --unsafe-perm -g npm-audit-html npm-audit-ci-wrapper sonar-scanner || cat /home/jenkins/.npm/_logs/*-debug.log
+RUN sonar-scanner || echo
```
Comment (pabrahamsson) [inline jenkins-slaves/jenkins-slave-npm/Dockerfile]: Please consolidate all `RUN` statements into one, similar to `Dockerfile.rhel7`.
Also no need for `USER root` here.

**Comment by pabrahamsson** [comment]
@InfoSec812 would you be able to address the above comments?

**Comment by InfoSec812** [comment]
@pabrahamsson & @sabre1041 Are we still waiting on something else here? I don't see any outstanding issues.

**Comment by sherl0cks** [comment]
Optimistically merging here. Feel free to open up issues or follow on PRs to improve.

**Pull request merged**